rts/motoko-rts/src/types.rs

```diff
@@ -374,6 +374,8 @@ pub const TAG_CONCAT: Tag = 25;
 pub const TAG_NULL: Tag = 27;
 pub const TAG_ONE_WORD_FILLER: Tag = 29;
 pub const TAG_FREE_SPACE: Tag = 31;
+pub const TAG_REGION_OBJ: Tag = 33;
+pub const TAG_REGION_VEC: Tag = 35;
```

Comment:
We would need to increase the constant value of `TAG_ARRAY_SLICE_MIN `to at least 36 as it reserves the range `>= TAG_ARRAY_SLICE_MIN` in the GC.

Reply:
Oh okay -- `37` then?  (Needs to be odd, it seems).